gulpfile.js/tasks/init.js

```diff
@@ -1,12 +1,12 @@
-var gulp = require('gulp')
-var log = require('fancy-log')
-var colors = require('ansi-colors')
+var gulp        = require('gulp')
+var log         = require('fancy-log')
+var colors      = require('ansi-colors')
 var projectPath = require('../lib/projectPath')
-var merge = require('merge-stream')
+var merge       = require('merge-stream')
```

Comment:
On our React work, anyway, we've stopped aligning requires. It tends to make diffs really noisy and it's frustrating when a really long import name forces everything over to the right.

Would it be possible to keep the old style?

Reply:
@davist11 voiced this complaint as well and i think it's valid.